cname_records.php

```diff
@@ -39,6 +39,11 @@
                 </div>
             </div>
             <div class="box-footer clearfix">
+              <strong>Note:</strong>
+                  <p>The target of a <code>CNAME</code> must be a domain that the Pi-hole knows the <code>A</code> value for, or is authoritative for that domain.</p>
+                  <p>Pi-hole will not do additional queries if you set the target to a domain that isn't already known. It just returns the information it knows at the time of the query.
+                    This results in certain limitations for <code>CNAME</code> targets, for instance, only <i>active</i> leases work as targets.</p>
+                    <p>Additionally, you can't <code>CNAME</code> external domains (<code>bing.com</code> to <code>google.com</code>) successfully as this would result in an invalid SSL certificate error.
```

Comment:
```suggestion
              <strong>Note:</strong>
                  <p>The target of a <code>CNAME</code> must be a domain that the Pi-hole already has in its cache or is authoritative for. This is an universal limitation of <code>CNAME</code> records.</p>
                  <p>The reason for this is that Pi-hole will not send additional queries upstream when serving <code>CNAME</code> replies. As consequence, if you set a target that isn't already known, the reply to the client may be incomplete. Pi-hole just returns the information it knows at the time of the query.
                    This results in certain limitations for <code>CNAME</code> targets, for instance, only <i>active</i> leases work as targets.</p>
                    <p>Additionally, you can't <code>CNAME</code> external domains (<code>bing.com</code> to <code>google.com</code>) successfully as this could result in invalid SSL certificate errors when the target server does not serve content for the requested domain.
```